### PR TITLE
Audio atom HTML, CSS and JS fixes

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Three changes:
- the timestamps are initialised in the template rather than in JS (it just looked odd)
- the play button had some padding that was visually not pleasing
- the duration is updated when the metadata arrives via the `durationchange` event

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
[In-Article Audio Atom ready to go live on one article (v2)](https://trello.com/c/sh46jpiC/598-in-article-audio-atom-ready-to-go-live-on-one-article-v2)

File: core/src/main/resources/lib/audio.fjs [hunk 1]
```diff
@@ -95,6 +88,11 @@ export default (componentType: ComponentType) => ({ ophan, dom, viewport }: Serv
     });
   };
 
+  const onDurationChange = () => {
```
Comment (JustinPinner) [inline core/src/main/resources/lib/audio.fjs]: Can we tie the scrubber seek calculation to this value too? It currently reads a .getAttribute('data-duration') which can be incorrect due to ads. It'd be nice to kill that issue now if we can.

Comment (regiskuckaertz) [reply core/src/main/resources/lib/audio.fjs]: Oooooh yeah you're right. will do 👍

Comment (regiskuckaertz) [reply core/src/main/resources/lib/audio.fjs]: Done. I will test as soon as the previewer has been approved.

**Comment by regiskuckaertz** [comment]
Tested in preview:

screen shot 2019-02-19 at 13 34 23

**Pull request merged**